docs/guide/catalog/product.md

```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
             - my_custom_product_type
```

Comment:
Should there be a minus before "my_custom_product_type"? 
`yaml` styling interprets it as substraction and colors the line red while we just said that we want to add this identifier to the config file.

Reply:
That's part of YAML's array syntax: https://www.w3schools.io/file/yaml-arrays/, see for example: https://github.com/ezsystems/ezcommerce-shop/blob/7169ce3dfbafca67f696b2009209d83414207bb8/src/Silversolutions/Bundle/EshopBundle/Resources/config/product_settings.yaml#L3